### PR TITLE
Rename Python files to have .py extension

### LINKED ISSUE #1003: Rename Python files to have .py extension
* [x] I have searched for duplicates or related issues

## Description

Avoid issues like https://github.com/freedomofpress/securedrop-workstation/pull/1002 by having every Python file have a .py extension so linters, editors, etc. flag it as a Python file. At install time (in rpm build), it can use a non-extension name.

### PR DESCRIPTION
**Pull request by micahflee**
## Status

Ready for review

## Description of Changes

Fixes #1003.

Changes proposed in this pull request:

I've renamed these files to end in `.py`:

- `dom0/remove-tags`
- `files/destroy-vm`
- `files/sdw-login`
- `files/sdw-notify`
- `files/sdw-updater`
- `scripts/configure-environment`

And I also searched the rest of the code for these filenames and updated the new filenames where appropriate.

The dom0 tests pass. Please review carefully though because I'm not sure what if anything I should be testing manually to make sure this didn't break anything.

## Testing

If you made non-trivial code changes, include a test plan and validated it for this PR.

## Deployment

Any special considerations for deployment? Consider both:

1. Upgrading existing pilot instances
2. New installs

### If you have made changes to the provisioning logic

- [x] All tests (`make test`) pass in `dom0`

### If you have added or removed files

- [x] I have updated `MANIFEST.in` and `rpm-build/SPECS/securedrop-workstation-dom0-config.spec`

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -4,8 +4,3 @@ max-line-length = 100
 extend-exclude = .venv
 filename =
     *.py
```
Comment (legoktm) [inline .flake8:6]: Can the explicit `*.py` be dropped now? I assume it's the default...

**Review by legoktm — CHANGES_REQUESTED** [review]
Thanks! Can we have the files be installed using the same names they currently have? For example, `sdw-admin.py` is installed as `sdw-admin` using:
```spec
install -m 755 files/sdw-admin.py %{buildroot}/%{_bindir}/sdw-admin
```

That way it's a no-op on a deployed system.

**Comment by micahflee** [comment]
Okay, I've made those changes and this should be good for another review @legoktm

File: pyproject.toml [hunk 2]
```diff
@@ -35,16 +35,14 @@ ignore_missing_imports = true
 scripts_are_modules = true
 files = [
     "*.py",
-    "dom0/remove-tags",
+    "dom0/remove-tags.py",
     "dom0/securedrop-login",
```
Comment (legoktm) [inline pyproject.toml:39]: This seems to be gone now, we can remove it in a follow-up.

**Review by legoktm — APPROVED** [review]
LGTM! I'm going to squash this and merge

**Review by legoktm — APPROVED** [review]

**Pull request merged**